backend/coins/btc/account.go

```diff
@@ -234,6 +236,31 @@ func (account *Account) gapLimits(
 	return limits, nil
 }
 
+// getMinRelayFeeRate fetches the min relay fee from the server and returns it. The value is cached
+// so that subsequent calls are instant. This is important as this function can be called many times
+// in succession when validating tx proposals.
+func (account *Account) getMinRelayFeeRate() btcutil.Amount {
+	unlock := account.RLock()
+	cached := account.minRelayFeeRate
+	unlock()
+	if cached != nil {
```

Comment:
Seems like this is cached for the whole lifetime of the app. If I keep it running for hours, wouldn't it be pretty much a stale fee amount?

Reply:
It is the min relay fee, it is a bitcoin core server config that does not change at runtime, so it is stable.

Different servers could have different min relay fees, which is why I reset the value on a re-connect.

Reply:
:+1: I misunderstood